ethcore/src/spec/spec.rs

```diff
@@ -186,17 +186,16 @@ fn run_constructors<T: Backend>(
 
 /// Parameters for a block chain; includes both those intrinsic to the design of the
 /// chain and those to be interpreted by the active chain engine.
+#[derive(Clone)]
```

Comment:
Do we use `spec.clone()` at all?
```suggestion
```

Reply:
I think this replaces the test-only `Clone` impl ([here](https://github.com/paritytech/parity-ethereum/pull/10948/files#diff-8c15897d65990126e87e06b4feefef7dL234))

Reply:
@dvdplm yeah, I saw that, but it code doesn't seem to use a clone in tests.
Otherwise, it can be replaced with 
```suggestion
#[cfg_attr(test, derive(Clone))]
```